cve_bin_tool/error_handler.py

```diff
@@ -154,6 +162,8 @@ def __exit__(self, exc_type, exc_val, exc_tb):
     FileNotFoundError: -3,
     InvalidCsvError: -4,
     InvalidJsonError: -4,
+    EmptyTxtError: -4,
```

Comment:
I wasn't sure about the error code so I left it as -4 since that made more sense to me than adding more codes

Reply:
I don't think negative error codes are a good idea, see #1135

Reply:
I think we'll resolve the error codes issue in a separate PR.  -4 should be fine for now.